core/lib/dal/migrations/20240307190434_change_to_u256.up.sql

```diff
@@ -0,0 +1,51 @@
+ALTER TABLE l1_batches ADD l1_gas_price_u256 numeric(80,0) NOT NULL DEFAULT 0;
+ALTER TABLE l1_batches ADD l2_fair_gas_price_u256 numeric(80,0) NOT NULL DEFAULT 0;
+
+ALTER TABLE miniblocks ADD l1_gas_price_u256 numeric(80,0) NOT NULL DEFAULT 0;
+ALTER TABLE miniblocks ADD l2_fair_gas_price_u256 numeric(80,0) NOT NULL DEFAULT 0;
+ALTER TABLE miniblocks ADD fair_pubdata_price_u256 numeric(80,0) NOT NULL DEFAULT 0;
+
+UPDATE l1_batches SET l1_gas_price_u256=l1_gas_price;
+UPDATE l1_batches SET l2_fair_gas_price_u256=l2_fair_gas_price;
+
+UPDATE miniblocks SET l1_gas_price_u256=l1_gas_price;
+UPDATE miniblocks SET l2_fair_gas_price_u256=l2_fair_gas_price;
+UPDATE miniblocks SET fair_pubdata_price_u256=fair_pubdata_price;
```

Comment:
This should be a programmatic migration, see https://github.com/matter-labs/zksync-era/tree/main/core/lib/dal#programmatic-migrations

Reply:
Got it, I'll remove the updates on the migration and instead add a task that every few seconds updates some of the rows in batches of around 5 or 10k until it's done. Is it reasonable to write the task in Rust/Tokio inside the operator or do you have another method of doing it? We couldn't find an example in the codebase.

Reply:
Check `fee_address_migration` as an example